docs/learn/wallets/iota-devnet-wallet.md

```diff
@@ -4,32 +4,19 @@ description: A Wallet for the current IOTA 2.o Development - and Testnet. Offers
 ---
 
 
-# Heading 1 (Page Title)
+# Devnet Wallet
```

Comment:
Devnet or DevNet? We should choose a consistent notation.

Reply:
Yeah i think Devnet.
```suggestion
# Devnet Wallet
```